init/systemd/go-fdo-server-manufacturer.service

```diff
@@ -1,25 +1,28 @@
+# Modifying this file in-place is not recommended, because changes
+# will be overwritten during package upgrades.  To customize the
+# behavior, run "systemctl edit go-fdo-server-manufacturer" to create
+# an override unit.
+#
+# For example, to pass additional command line options to the server
+# at startup create an override unit (as is done by systemctl edit)
+# and enter the following:
+#
+#	[Service]
+#	Environment=OPTIONS="--log-level debug"
+
 [Unit]
 Description=Go FDO manufacturer server
+Requires=go-fdo-server-init.service
+After=go-fdo-server-init.service
 After=network.target
 
 [Service]
 Type=simple
 PrivateDevices=yes
 CacheDirectory=go-fdo-server-manufacturer
 StateDirectory=go-fdo-server-manufacturer
-ExecStartPre=/usr/share/go-fdo-server/generate-manufacturer-certs.sh
-ExecStart=/bin/sh -c 'exec /usr/bin/go-fdo-server \
-                                 --db-type ${DATABASE_TYPE} \
-                                 --db-dsn ${DATABASE_DSN} \
-                                 manufacturing ${LISTEN_IP}:${LISTEN_PORT} \
-                                 --manufacturing-key ${MANUFACTURER_KEY} \
-                                 --owner-cert ${OWNER_CRT} \
-                                 --device-ca-cert ${DEVICE_CA_CRT} \
-                                 --device-ca-key ${DEVICE_CA_KEY} \
-                                 ${ADDITIONAL_OPTS}'
+ExecStart=/bin/sh -c 'exec /usr/bin/go-fdo-server manufacturing $OPTIONS'
```

Comment:
Maintaining environment variables in the `ExecStart` does not make sense to me if we drop `sysconfig` files completely. To change the command line options we need to create a drop-in file:
```
[Service]
Environment=OPTIONS="--log-level debug"
```

which is the same as:
```
[Service]
ExecStart= 
ExecStart=/usr/bin/go-fdo-server --log-level debug manufacturing
```

And run `systemctl daemon-reload` in both cases.
I personally find it cleaner and simpler just to override the full command which doesn't require the execution of a shell to expand the variables either.